pom.xml

```diff
@@ -2374,6 +2374,16 @@
             <scope>test</scope>
             <version>${version.infinispan}</version>
          </dependency>
+         <dependency>
+            <groupId>org.infinispan</groupId>
+            <artifactId>infinispan-commons-base</artifactId>
```

Comment:
where is commons-base defined?

Reply:
That was a leftover from a previous iteration. Removed